sys/random/Makefile

```diff
@@ -11,4 +14,28 @@ ifneq (,$(filter prng_tinymt32,$(USEMODULE)))
   DIRS += tinymt32
 endif
 
+ifeq (,$(RIOT_CI_BUILD))
+  # random seed value newly generated on each build
+  RANDOM_SEED := $(shell $(RIOTTOOLS)/randhex/randhex.py 32)
+else
+  # hardcoded canary value to not break CI test cache
+  RANDOM_SEED := 0x1
+endif
+
+# This has to be kept in sync with sys/random/init.c
+ifneq (,$(filter puf_sram periph_hwrng periph_cpuid,$(USEMODULE)))
```

Comment:
Just as a heads up: Changed this check, as it seems checking `FEATURES_USED` is not sufficient, instead `USEMODULE` needs to be used for this check.